assets/app/scripts/controllers/overview.js

```diff
@@ -384,7 +416,114 @@ angular.module('openshiftConsole')
       });
     });
 
+    /*
+     * The topology graph wants unique identifiers.
+     *
+     * One of the warts on the kubernetes/openshift API is that there
+     * is no unique global identifier per resource. We construct them here from
+     * Kind/metadata.name ... which is good enough in a single namespace.
+     */
+    function makeId(name, kind) {
```

Comment:
> Please open a Kubernetes issue for that if one is not there already.

@derekwaynecarr Will do.

Thinking about it more ... for *this* case ```metadata.selfLink``` probably does what we want. It's not unique across different objects with the same name/namespace but for the topology graph that doesn't matter.

I'll update the code to use that, unless there are objections.